Assets/Scripts/UI/MainMenuScript.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using TMPro;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace UI
+{
+    public class MainMenuScript : Menu
+    {
+        public void PlayGame()
+        {
+            SceneManager.LoadScene(1);
+        }
+
+        public void QuitGame()
+        {
+        #if UNITY_EDITOR
+            UnityEditor.EditorApplication.isPlaying = false;
+        #else
+            Application.Quit();
+        #endif
```

Comment:
Never knew this is how to exit a game in unity while its in the editor. my "final quiz" last semester kept crashing when i did quit and i was like bruh wtf

Reply:
i know right this is @fmichaut-dev's brain in action